src/sapling/saplingscriptpubkeyman.cpp

```diff
@@ -447,6 +447,23 @@ libzcash::SaplingPaymentAddress SaplingScriptPubKeyMan::GenerateNewSaplingZKey()
     return xsk.DefaultAddress();
 }
 
+void SaplingScriptPubKeyMan::GetConflicts(const CWalletTx& wtx, std::set<uint256>& result) const
+{
+    AssertLockHeld(wallet->cs_wallet);
+    std::pair<TxNullifiers::const_iterator, TxNullifiers::const_iterator> range_o;
+
+    for (const SpendDescription& spend : wtx.sapData->vShieldedSpend) {
+        uint256 nullifier = spend.nullifier;
```

Comment:
(11a58e6f6f461670d7593dedf36996c3a8004a3e)
nit: use const reference or directly call `mapTxSaplingNullifiers.count(spend.nullifier)` in the next lines.